src/cbmc/all_properties.cpp

```diff
@@ -56,8 +57,9 @@ safety_checkert::resultt bmc_all_propertiest::operator()()
 
   solver.set_message_handler(get_message_handler());
 
-  // stop the time
-  absolute_timet sat_start=current_time();
+  std::chrono::time_point<std::chrono::steady_clock, std::chrono::seconds>
+    sat_start = std::chrono::time_point_cast<std::chrono::seconds>(
+      std::chrono::steady_clock::now());
```

Comment:
Would it be worthwhile keeping a `time_stopping.h` (but really just the header) with

````c++
#include <chrono>

using absolute_timet = 
  std::chrono::time_point<std::chrono::steady_clock, std::chrono::seconds>;

absolute_timet current_time()
{
  return
    std::chrono::time_point_cast<std::chrono::seconds(
      std::chrono::steady_clock::now());
}
````